exec/cnex/cnex.c

```diff
@@ -325,6 +344,41 @@ void exec_rtl_raiseException(TExecutor *self, const char *name, const char *info
     string_freeString(n);
 }
 
+struct Ne_MethodTable ExtensionMethodTable = {
```

Comment:
There's no reason to define this table in *this* source module. If you put it in `extension.c`, then you don't have to list all these functions in `extension.h`.

Reply:
Moved to extension.c, and eliminated extension.h.